### DISCUSSION TITLE
Trust Wallet  issue with wallet connect

### DISCUSSION THREAD

**Discussion by arbaz-meo** [WalletConnect/walletconnect-monorepo#2553] [Developer Support]
Trust wallet work fine when we launch universal link ( that's creating through sign client ) through mobile app (native apps ) or browser but its not open the authentication pop up when we launch  universal link through games (expect unity ) while its opening the wallet and redirecting similarly

**Comment by arbaz-meo** [comment]
wallet connect pop up opens in background but its already redirect to the game before open pop up .

**Reply by finessevanes** [reply]
Are you able to share what device you're using as well as the version for Trust?

**Reply by arbaz-meo** [reply]
trust  wallet version 7.23 and using  oppo Reno 8 , svmsung galaxy A52  and also test on huawei having the same issue

**Reply by arbaz-meo** [reply]
@finessevanes  its open the pop up in  trust wallet if we disable the security level of trust wallet  from setting .
